### PR TITLE
Implemented caching with webpack

### LINKED ISSUE #59: Disable caching in browser
Should disable caching old builds to make sure user's don't have to constantly clear cache to view changes.

https://stackoverflow.com/questions/40243633/disable-nginx-cache-for-javascript-files

https://stackoverflow.com/questions/44605647/long-term-caching-with-webpack-chunkhash-and-cache-control

### PR DESCRIPTION
**Pull request by MohamedBeydoun**
Resolves #59 

Added timestamp to output files during build

### CODE REVIEW

**Review by ValerianClerc — DISMISSED** [review]
You know that I don't know anything about NGINX, so I'll just throw in this approval.

**Comment by imad-ant** [comment]
I'm not sure I agree with this change - can we not update webpack configuration to hash files? That would allow users to cache but force retrieval of resources on new builds/releases

File: public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html data-build-timestamp-utc="<%= new Date().toISOString() %>" lang="en">
```
Comment (ValerianClerc) [inline public/index.html:2]: Says “UTC” in the attribute name, but JavaScript’s Date object uses your local time zone. You should convert this to UTC.

**Review by ValerianClerc — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (MohamedBeydoun) [reply public/index.html:2]: ISO timestamps show time with an offset from UTC based on ur timezone. So after the conversion, the time will be in UTC.

Comment (ValerianClerc) [reply public/index.html:2]: Either remove “utc” from the attribute title, or explicitly convert to UTC in the JS imo

**Review by ValerianClerc — APPROVED** [review]

**Review by imad-ant — APPROVED** [review]
Lgtm!

**Pull request merged**